docs/guides/02_page_layer_widgets.md

```diff
@@ -214,7 +214,7 @@ Summary:
     2. Exports the ID as a constant.
 - The layer must be added to the application layers array.
 - You need to add the source and the layer to the store.
-- `cartoLayerProps` are required.
```

Comment:
Here I think it is talking about the variable, not the hook. But nevermind, we're gonna remove the guides from the template soon